torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -55,7 +64,7 @@ class PowerSGDState(object):
 
     To tune ``start_powerSGD_iter``, we suggest to start with 10% of total training steps, and increase it until a satisfactory accuracy is reached.
 
-    3. ``min_compression_rate`` is the minimal compression rate required when a layer is compressed. Due to the computation overheads incurred by the compression, a tensor is worth compressing only if there can be sufficient saving in bandwidth, where `` (num_rows + num_cols) * matrix_approximation_rank < num_rows * num_cols * min_compression_rate``. If the specified compression rate threshold cannot be satisfied, the tensor will be directly allreduced without compression.
+    3. ``min_compression_rate`` is the minimum compression rate required when a layer is compressed. Due to the computation overheads incurred by the compression, a tensor is worth compressing only if there can be sufficient saving in bandwidth, where `` (num_rows + num_cols) * matrix_approximation_rank * min_compression_rate < num_rows * num_cols``. If the specified compression rate threshold cannot be satisfied, the tensor will be directly allreduced without compression.
```

Comment:
nit: do you want to add the "min_compression_rate = uncompressed size/compressed size" in the doc as well?

Reply:
I mentioned the requirement of `` (num_rows + num_cols) * matrix_approximation_rank * min_compression_rate < num_rows * num_cols`` in the docstring, which should be equivalent.